download-packages-popularity.ts

```diff
@@ -87,7 +87,7 @@ export const packages = [
     packageName: 'r-assign',
   },
   {
-    name: 'rescript-schema',
+    name: 'sury',
     packageName: 'rescript-schema',
```

Comment:
Kept it as `rescript-schema` for now. I'll update it later when users migrate to `sury`

Reply:
mhh okay.

From my POV it maybe would have been cleaner then to just add sury as a new library to the benchmark and remove rescript-schema once the migration is done.

Could you maybe please add a comment though in case someone wonders about the naming mismatch :grin: ? thx!